python/mlc_chat/rest.py

```diff
@@ -63,7 +63,7 @@ def _parse_args():
     args.add_argument(
         "--model", type=str, default="vicuna-v1-7b"
     )
-    args.add_argument("--artifact-path", type=str, default="dist")
+    args.add_argument("--artifact-path", type=str, default="../dist")
```

Comment:
I think this might not be a desired behavior because if make wrap `mlc_chat` as a package, then the user might execute the module under another folder.

Reply:
I agree we should use the project root as the root here. It can be invoked via 
```
PYTHONPATH=python python -m mlc_chat.rest
```

Reply:
Got it, reverted this in https://github.com/mlc-ai/mlc-llm/pull/357/commits/c27402c1edb79778144440adc196ef5187373f12.